zerver/lib/test_classes.py

```diff
@@ -543,9 +543,11 @@ def unsubscribe(self, user_profile: UserProfile, stream_name: Text) -> None:
     # Subscribe to a stream by making an API request
     def common_subscribe_to_streams(self, email: Text, streams: Iterable[Text],
                                     extra_post_data: Dict[str, Any]={}, invite_only: bool=False,
+                                    stream_admins: List[Text]=[],
```

Comment:
We should the clients pass in a list of user_ids, not emails, for this.  Emails are super painful to work with, since they can change and they can apply to multiple orgs.